api_tests/nodes/views/test_node_relationship_institutions.py

```diff
@@ -28,137 +31,166 @@ def read_contrib_institution(self):
         return InstitutionFactory()
 
     @pytest.fixture()
-    def resource_factory(self):
-        return NodeFactory
-
-    @pytest.fixture()
-    def make_resource_url(self):
-        def make_resource_url(node):
-            return f'/{API_BASE}nodes/{node._id}/relationships/institutions/'
-        return make_resource_url
+    def resource_url(self, node):
+        return f'/{API_BASE}nodes/{node._id}/relationships/institutions/'
 
     @pytest.fixture()
     def user(self, institution_one, institution_two):
-        user = AuthUserFactory()
```

Comment:
not a good idea to use a common fixture name here, that's what Pycharm tells me anyway.